tribuo-classification/client/src/main/java/com/r3/conclave/sample/client/Client.java

```diff
@@ -0,0 +1,133 @@
+package com.r3.conclave.sample.client;
+
+import com.r3.conclave.client.EnclaveConstraint;
+import com.r3.conclave.client.InvalidEnclaveException;
+import com.r3.conclave.common.EnclaveInstanceInfo;
+import com.r3.conclave.mail.Curve25519PrivateKey;
+import com.r3.conclave.mail.EnclaveMail;
+import com.r3.conclave.mail.PostOffice;
+import com.r3.conclave.sample.common.InputData;
+import org.tribuo.MutableDataset;
+import org.tribuo.classification.LabelFactory;
+import org.tribuo.data.csv.CSVLoader;
+import org.tribuo.datasource.ListDataSource;
+import org.tribuo.evaluation.TrainTestSplitter;
+
+import java.io.*;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.nio.file.Paths;
+import java.security.PrivateKey;
+import java.util.UUID;
+
+/**
+ * The clients for this application will be different doctors/hospitals. They will upload files containing breast cancer dataset.
+ * The client will serialise this dataset and send it to the enclave. The enclave will load the model, train the model with this
+ * data. The client can send test data to test this trained model. If the results of this testing are as expected/correct, this
+ * trained model can be used to make predictions of a given input tumor.
+ *
+ */
+public class Client {
+    private static final String TRAIN = "TRAIN";
+
+    public static void main(String[] args) throws InterruptedException, IOException, InvalidEnclaveException {
+
+        if (args.length == 0) {
+            System.err.println("Please pass [TRAIN/EVALUATE] [FILENAME] [CONSTRAINT] containing input dataset to train/test the model.");
```

Comment:
I think the message is now confusing with the introduction of `[CONSTRAINT]`. The last part: "containing input dataset to train/test the model" refers to the filename. You might want to use `<paramName>`, which means mandatory parameter, rather than `[]`, which means optional.